src/olgm/herald/feature.js

```diff
@@ -207,13 +203,23 @@ olgm.herald.Feature.prototype.setVisible = function(value) {
   }
 };
 
+/**
+ * @private
+ * @return {ol.geom.Geometry} the feature's geometry
+ */
+olgm.herald.Feature.prototype.getGeometry_ = function() {
+  var geometry = this.feature_.getGeometry();
+  olgm.asserts.assert(
+      geometry !== undefined, 'Expected feature to have geometry');
+  return /** @type {ol.geom.Geometry} */ (geometry);
+};
+
```

Comment:
Suggestion (feel free to comment)

How about we move this method to `olgm.feature.getGeometry`, which would be located in `./src/feature.js`, as a utility method to get a geometry from a feature which is expected to have a geometry?